hemi/configure.h

```diff
@@ -1,7 +1,7 @@
 ///////////////////////////////////////////////////////////////////////////////
-// 
```

Comment:
Changes to configure.h seem to be only whitespace changes. Happy to accept pull requests for fixes to incorrect spacing, but let's keep that as a separate pull request. Can you eliminate all whitespace-only changes from this PR to make it easier to review?